lld/MachO/SyntheticSections.cpp

```diff
@@ -865,12 +910,13 @@ void ObjCStubsSection::setUp() {
                                 /*align=*/target->wordSize);
   in.objcSelrefs->live = true;
 
-  for (size_t i = 0, n = offsets.size(); i < n; ++i) {
+  for (const auto &[methname, idxOffsetPair] : methnameToidxOffsetPair) {
+    const auto &[idx, offset] = idxOffsetPair;
     in.objcSelrefs->relocs.push_back(
         {/*type=*/target->unsignedRelocType,
          /*pcrel=*/false, /*length=*/3,
-         /*offset=*/static_cast<uint32_t>(i * target->wordSize),
-         /*addend=*/offsets[i] * in.objcMethnameSection->align,
+         /*offset=*/static_cast<uint32_t>(idx * target->wordSize),
+         /*addend=*/offset * in.objcMethnameSection->align,
```

Comment:
Multiplying by `align` seems kind of weird. It just happens to equal `target->wordSize`, but we should just reference wordSize directly

(I know this is existing code)

Reply:
I think `in.objcMethnameSection->align` is actually 1. yeah, it's a bit odd to multiply this, so I would just drop it.